common/metrics/defs.go

```diff
@@ -2670,7 +2673,9 @@ const (
 	AsyncWorkflowFailureCorruptMsgCount
 	AsyncWorkflowFailureByFrontendCount
 	AsyncWorkflowSuccessCount
-
+	DiagnosticsWorkflowCount
```

Comment:
nit: maybe DiagnosticsWorkflowStartedCount
I was not sure if it was started or completed until I checked further down the code

Reply:
thanks. updated it